### PR TITLE
20055 - Updating nuxt-ui, pinia, vite vitetest

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:20055*https://github.com/bcgov/entity/issues/20055 

*Description of changes:*
- Update nuxt/ui to newest version
- update pinia to newest version
- update vitetest to newest version
- update vite to newest version

ticket to fix commented out tests #20310

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-139-ghhci77q.web.app

File: btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue [hunk 1]
```diff
@@ -98,6 +99,8 @@ const canadaPostApiKey = runtimeConfig.public.addressCompleteKey
 const suggestedAddresses: Ref<Array<CanadaPostApiFindResponseItemI>> = ref([])
 // @ts-ignore
 const line1: Ref<string> = ref(props.modelValue)
+const val = ref()
+val.value = { id: 'empty' }
```
Comment (kialj876) [inline btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue:103]: left this in by accident I think?

File: btr-web/btr-common-components/vitest.config.mts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { fileURLToPath } from 'node:url'
-import { defineVitestConfig } from 'nuxt-vitest/config'
-
+import { defineVitestConfig } from '@nuxt/test-utils/config'
+//
```
Comment (kialj876) [inline btr-web/btr-common-components/vitest.config.mts:3]: can rmv this too

File: btr-web/btr-main-app/utils/breadcrumbs.ts [hunk 3]
```diff
@@ -1,18 +1,21 @@
 export function getBcrosHomeDashboardCrumb (): BreadcrumbI {
-  const { t } = useI18n()
+  const { $i18n } = useNuxtApp()
```
Comment (kialj876) [inline btr-web/btr-main-app/utils/breadcrumbs.ts:2]: I think this is a little nicer, but up to you:
`const t = useNuxtApp().$i18n.t`

File: btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue (hunk 1 above)
Comment (kialj876) [reply btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue:103]: Oh nvm combobox uses this or were you just experimenting with something? Doesn't look like it changes at all

**Review by kialj876 — APPROVED** [review]
Hey looks good to me, nice work! I just have a few minor comments / questions

File: btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue (hunk 1 above)
Comment (hfekete) [reply btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue:103]: This one is required to have the value of the combobox refreshed first time. If you pass the value as prop. 
In short, its a workaround for how the headless UI keeps value and display value in input box :/

File: btr-web/btr-main-app/utils/breadcrumbs.ts (hunk 3 above)
Comment (hfekete) [reply btr-web/btr-main-app/utils/breadcrumbs.ts]: Updated the code to the nicer suggestion you have.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-139-ghhci77q.web.app

**Pull request merged**